docs/app/Examples/elements/Header/Content/HeaderImageExample.js

```diff
@@ -0,0 +1,16 @@
+import faker from 'faker';
+import React, {Component} from 'react';
+import {Header, Image} from 'stardust';
+
+export default class HeaderImageExample extends Component {
+  render() {
+    const photo = <Image className='circular' src={faker.image.city()} />;
+    return (
+      <Header.H2 image={photo}>
+        <div className='content'>
```

Comment:
It would be preferable to abstract away this `content` div.  That way the user just has to:

```jsx
<Header.H2>
  Welcome to Our City!
</Header.H2>
```

This would also be consistent with how we handled other components.